docs/kyma/docs/03-03-testing.md

```diff
@@ -3,11 +3,15 @@ title: Testing Kyma
 type: Details
 ---
 
-For testing, the Kyma components use the Helm test concept. Place your test under the `templates` directory as a Pod definition that specifies a container with a given command to run.
+For testing, the Kyma components use the [Octopus](http://github.com/kyma-incubator/octopus). 
+Octopus is a testing framework that allows you to run tests defined as Docker images on a running cluster.
+Octopus uses two types of Custom Resource Definitions:
+- TestDefinition defines your test as a Pod specification.
+- ClusterTestSuite defines a suite of tests to execute and how to execute them.
 
-## Add a new test
+Place your TestDefinition in your chart.
 
-The system bases tests on the Helm broker concept with one modification: adding a Pod label. Before you create a test, see the official [Chart Tests](https://github.com/kubernetes/helm/blob/release-2.10/docs/chart_tests.md) documentation. Then, add the `"helm-chart-test": "true"` label to your Pod template.
+## Add a new test
 
 See the following example of a test prepared for Dex:
```

Comment:
Let's make this section more generic. Start with a more general sentence, such as:
```
To add a new test, create a yaml file under the `templates/tests` directory. See the exemplary test directory prepared for Dex:
```

... and here you go with the directory structure.